templates/openstackplaybookgenerator/config/fencing.yaml

```diff
@@ -13,6 +13,6 @@ parameter_defaults:
           namespace: {{ $fencingConfig.Namespace }}
           ssl_insecure: true
           power_timeout: 30
-          kubeconfig: /home/cloud-admin/kubeconfig
+          kubeconfig: /mnt/kubeconfig/kubeconfig
```

Comment:
not related to this file, but to the kubeconfig in general:
* what are the permissions to the kubeconfig? we need to make sure that only root can access it so that it can not be used by any other user. maybe we just place it in /root/kubeconfig/kubeconfig ?
* how does pacemaker behave if the config is no longer accessible for whatever reason. should we copy the config on boot instead of direct use of the secret mount?

Reply:
We can certainly mount the kubeconfig elsewhere.  And yes, we can also tighten the filesystem perms to `0600` for `root`-only, or something like that.

If the kubeconfig is not available, the call to `/usr/sbin/fence_kubevirt` will fail.  In terms of what you'd see in the OSP cloud, `pcs status` would report a failed fencing action.

Reply:
as discussed on slack, that was the idea on this to prevent any fencing action because the kubeconfig is not available.